nodejs-bootstrap-datepicker/nodejs-bootstrap-datepicker.spec

```diff
@@ -0,0 +1,45 @@
+%global npm_name bootstrap-datepicker
+%global enable_tests 1
+
+Name: nodejs-%{npm_name}
+Version: 1.7.1
```

Comment:
In doing some Rails 5.1 testing (cause I need this patch) I ran into patternflly has a requirement of ~1.6.4 which translates into < 1.7.0 within the patternfly RPM spec so I think we need to lower this version included?

Reply:
I assume you found that with a repoclosure check. It'd be good to have that here as well.